docs/src/components/GuidesNavigationFooter.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import { NavLink } from 'react-router-dom'
+import { Button, Flex, Divider } from '@stardust-ui/react'
+
+type PageDescriptor = {
+  name: string
+  url: string
+}
+
+type GuidesNavigationFooterProps = {
+  previous?: PageDescriptor
+  next?: PageDescriptor
+}
+
+const GuidesNavigationFooter: React.FC<GuidesNavigationFooterProps> = ({ previous, next }) => (
+  <>
+    <br />
+    <Divider size={1} />
+    <br />
+    <Flex gap="gap.small">
+      {previous && (
+        <Button
+          as={NavLink}
+          content={previous.name}
+          icon="arrow left"
+          iconPosition="before"
+          primary
+          to={previous.url}
+        />
+      )}
+      {next && (
+        <Button
+          as={NavLink}
```

Comment:
Actually, can be just `Link` because `NavLink` offers additional styling that will never work: https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md